docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -162,6 +162,22 @@ children:
     url: Decorators.html
     output: 'web, pdf'
```

Comment:
After this line you need to add a `children: ` to make sure Jekyll knows the pages indented below are child pages. It's also why `verify-docs` is failing on Travis.

Reply:
fixed, thanks